delorean-user-data.txt

```diff
@@ -280,18 +280,19 @@ write_files:
     permissions: 0755
 
 bootcmd:
-#  - echo "" >> /etc/ssh/sshd_config
-#  - echo "Port 330" >> /etc/ssh/sshd_config
-#  - systemctl restart sshd
+  - echo "" >> /etc/ssh/sshd_config
+  - echo "Port 3300" >> /etc/ssh/sshd_config
+  - echo "Port 22" >> /etc/ssh/sshd_config
```

Comment:
We've changed the fail2ban port to 3300 if listening on port 22 also will it need to be a list?